src/include/OpenImageIO/imagebufalgo.h

```diff
@@ -680,14 +688,14 @@ bool OIIO_API resize(ImageBuf &dst, const ImageBuf &src, KWArgs options = {},
 /// corresponding portion of `src` (mapping such that the "full" image
 /// window of each correspond to each other, regardless of resolution).  If
 /// `dst` is not yet initialized, it will be sized according to `roi`.
-/// 
+///
```

Comment:
Can you please remove all spurious formatting changes that are not directly related to the PR? Thanks.

Reply:
Yeah, sorry about that. It seems removing trailing white space on save is a default feature in modern IDEs.
I'll get to it sthe next week.
That said: wouldn't it make sense to remove trailing white space in the entire code base to save anyone from going though this on a PR?
I guess I'm asking if you'd consider a separate PR for just that?

Reply:
Yes, certainly. I just want code changes to not have formatting changes to any code that's not changing. A PR that is nothing but formatting changes is fine as long as no code changes. (Among other things, that allows us to add the formatting-only commit hashes to `.git-blame-ignore-revs` so that they don't change the apparent history or authorship of any actual code.)